src/computation/computation.graphql

```diff
@@ -14,30 +14,24 @@ type CourseEntry {
 
 input CourseRecommendationRequest {
     variant: String!
-    semesterKey: SemesterKeyInput!
-	selectedCourse: [CourseKeyInput!]! 
-}
-
-input SemesterKeyInput {
-    studyProgram: String!
-	semester: String!
-    academicYear: String!
+    semesterKey: CourseGroupInput!
+    selectedCourses: [CourseKeyInput!]! 
 }
 
 input CourseKeyInput {
     courseNo: String!
-	semesterKey: SemesterKeyInput!
+    semesterKey: CourseGroupInput!
```

Comment:
`courseGroup` instead of `semesterKey` to go along the class name?

Reply:
applies to all `semesterKeys` in the schemas

Reply:
ohhhh wait. I don't see this comment

Reply:
well, probably need to keep it that way, because this specific name is coupled in the computation backend.